### DISCUSSION TITLE
Can motion add rtsp stream in wifi ip camera?

### DISCUSSION THREAD

**Discussion by hwypengsir** [Motion-Project/motion#1706] [Q&A]
I can make usb camera to a mjpeg stream on 127.0.0.1:8081.In my home,there is a wifi ip camera,  i can see the video with ffplay or vlc:

    ffplay  rtsp://admin:xxxx@192.168.31.97:554/live

Is there a way to add it with motion?Let motion manage the stream?

**Comment by tosiara** [comment]
Specify in config:
`netcam_url rtsp://admin:xxxx@192.168.31.97:554/live`
